networkx/algorithms/dominance.py

```diff
@@ -50,9 +50,8 @@ def immediate_dominators(G, start):
 
     References
     ----------
-    .. [1] K. D. Cooper, T. J. Harvey, and K. Kennedy.
-           A simple, fast dominance algorithm.
-           Software Practice & Experience, 4:110, 2001.
+    .. [1] Cooper, Keith D., Harvey, Timothy J. and Kennedy, Ken. "A simple, fast dominance algorithm." (2006).
+https://hdl.handle.net/1911/96345
```

Comment:
I think the linting tool wants this to be wrapped something like this (and for the other function too).
```suggestion
    .. [1] Cooper, Keith D., Harvey, Timothy J. and Kennedy, Ken,
           "A simple, fast dominance algorithm." (2006).
           https://hdl.handle.net/1911/96345
```

Reply:
Shouldn't it always be 4-space indents?

```suggestion
    .. [1] Cooper, Keith D., Harvey, Timothy J. and Kennedy, Ken.
        "A simple, fast dominance algorithm." (2006).
        https://hdl.handle.net/1911/96345
```

Reply:
I don't think "always" is a good rule for most things. :)

Sphinx works either way for the html, and lining up the indents looks better when reading the ascii doc_string as part of the file. I would bet it isn't consistent throughout the library and that's ok. But that goals should be:
- readable inline doc_string while reading the code, and 
- nice html output when processed by Sphinx.